go/datas/http_chunk_store_test.go

```diff
@@ -151,28 +151,32 @@ func (suite *HTTPChunkStoreSuite) TestPutChunk() {
 	suite.True(suite.http.Has(c.Hash()))
 
 	suite.True(suite.http.Commit(hash.Hash{}, hash.Hash{}))
-	suite.Equal(1, suite.serverCS.Writes)
+	suite.Equal(2, suite.serverCS.Writes)
```

Comment:
This seems right to me but it means that we are now doing more writes. TestPutChunksInOrder also changed similarly below.